site/content/docs/0.0/migration-from-boosted.md

```diff
@@ -56,25 +56,38 @@ Technically, it means that you can get rid of the following things:
 - <details class="mb-2">
     <summary><span class="badge text-bg-warning">Warning</span> Dropped deprecated Sass variables:</summary>
     <ul>
-      <li><code>$btn-close-white-color</code></li>
+      <li><code>$boosted-prefix</code></li>
+      <li><code>$boosted-variable-prefix</code></li>
+      <li><code>$btn-close-white-active-border-color</code></li>
+      <li><code>$btn-close-white-active-color</code></li>
       <li><code>$btn-close-white-bg</code></li>
       <li><code>$btn-close-white-border-color</code></li>
-      <li><code>$btn-close-white-hover-color</code></li>
-      <li><code>$btn-close-white-active-color</code></li>
-      <li><code>$btn-close-white-active-border-color</code></li>
+      <li><code>$btn-close-white-color</code></li>
       <li><code>$btn-close-white-disabled-color</code></li>
+      <li><code>$btn-close-white-hover-color</code></li>
+      <li><code>$form-star-focus-box-shadow</code></li>
+      <li><code>$form-star-focus-color</code></li>
+      <li><code>$form-star-focus-color-dark</code></li>
+      <li><code>$form-star-focus-outline</code></li>
+      <li><code>$form-star-focus-outline-dark</code></li>
+      <li><code>$footer-nav-link-font-weight</code></li>
```

Comment:
Footer should come before form.

Reply:
Good catch, fixed at https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2638/commits/e5935cb42bdb282e9b2a664391b37ac632c20296